Toolchain/BuildIt.sh

```diff
@@ -64,7 +64,11 @@ mkdir -p "$DIR/Build/binutils"
 mkdir -p "$DIR/Build/gcc"
 
 if [ -z "$MAKEJOBS" ]; then
-    MAKEJOBS=$(nproc)
+    if ["$(uname -s)"=="Linux"]; then
```

Comment:
This line needs to read `if [ "$(uname -s)" = "Linux" ]; then` (all whitespace required) in order to work correctly